variables.tf

```diff
@@ -21,6 +21,24 @@ variable "name" {
   nullable    = false
 }
 
+variable "bypass_platform_safety_checks_on_user_schedule_enabled" {
+  type        = bool
+  description = "(Optional) Specifies whether to skip platform scheduled patching when a user schedule is associated with the VM. Only valid if patch_mode is AutomaticByPlatform."
+  default     = false
+  nullable    = true
+}
+
+variable "reboot_setting" {
+  type        = string
+  description = "(Optional) Specifies the reboot setting for platform scheduled patching. Possible values are Always, IfRequired and Never. Only valid if patch_mode is AutomaticByPlatform."
+  default     = null
+  nullable    = true
```

Comment:
We can omit this `nullable` since it defaults to `true`.

Reply:
Please remove this `nullable = true`, thanks!

Reply:
removed nullable, but it is not a boolean. Should we have a default?